src/index.js

```diff
@@ -0,0 +1,177 @@
+const state = {
+  temperature: 70,
+  cityName: 'Seattle',
+  skyValue: 'Sunny',
+};
+
+const increaseTemp = (event) => {
+  const tempDisplay = document.querySelector('#temperature');
+  const landScape = document.querySelector('#landscape');
+
+  state.temperature++;
+  tempDisplay.textContent = `${state.temperature}`;
+
+  changeTemperatureColor(state.temperature, tempDisplay);
+  changeLandScape(state.temperature, landScape);
+};
+
+const decreaseTemp = (event) => {
+  const tempDisplay = document.querySelector('#temperature');
+  const landScape = document.querySelector('#landscape');
+
+  state.temperature--;
+  tempDisplay.textContent = `${state.temperature}`;
+
+  changeTemperatureColor(state.temperature, tempDisplay);
+  changeLandScape(state.temperature, landScape);
+};
```

Comment:
There is a lot of repeated code in these two functions and in `findTemp` - I recommend thinking about how you could refactor to DRY the code.